src/components /Reviews/Reviews.jsx

```diff
@@ -1,25 +1,33 @@
 import { useEffect, useState } from "react";
 import { getReviews } from "../../utils/axios";
 import Review from "../Review/Review";
+import { useLocation, useParams, useSearchParams } from "react-router-dom";
+import Categories from "../Categories/Categories";
 
 function Reviews() {
   const [reviews, setReviews] = useState([]);
   const [loading, setLoading] = useState(false);
+  const { search } = useLocation();
+
+  const { category_name } = useParams();
+  console.log(category_name);
   useEffect(() => {
-    setLoading(true);
-    getReviews().then((reviews) => {
+    getReviews(category_name).then((reviews) => {
       setReviews(reviews);
       setLoading(false);
```

Comment:
we need setLoading(true) at the start of the useEffect otherwise we are never rendering anything with 'loading ?'.  Currently I don't see a loading message when I click around